sdk/src/crypto/raw_signature/signer.rs

```diff
@@ -177,6 +177,13 @@ impl From<crate::crypto::raw_signature::openssl::OpenSslMutexUnavailable> for Ra
     }
 }
 
+/// Convert JSON-encoded PEM data (with \n) to proper PEM format
+fn fix_json_pem(data: &[u8]) -> Vec<u8> {
+    String::from_utf8_lossy(data)
+        .replace("\\n", "\n")
```

Comment:
Essentially that's to handle potential formatting of the settings file, since for readability we will have newlines.